### PR TITLE
Updated readme to v3

### PR DESCRIPTION
**Pull request by Geet06**
Removed excess code and added note to use third-party plugins with webpack

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,19 +74,7 @@ aurelia.use.plugin('aurelia-froala-editor');
 <froala-editor></froala-editor>
 ```
 
-- In `aurelia_project/aurelia.json` file set the builder loader plugins stub to `false`
```
Comment (kapil2704) [inline README.md]: why are we removing this ?

File: README.md [hunk 2]
```diff
@@ -121,39 +111,6 @@ aurelia.use.plugin('aurelia-froala-editor');
 }
 ```
```
Comment (kapil2704) [inline README.md]: @Geet06 why we are removing this.

File: README.md (hunk 1 above)
Comment (vaibhavjain3) [reply README.md]: @kapil2704 please check line 77 below

Comment (Geet06) [reply README.md]: @kapil2704 please check line 77 below.

**Pull request merged**